packages/fiori/src/IllustratedMessage.js

```diff
@@ -9,7 +9,6 @@ import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
 import IllustratedMessageTemplate from "./generated/templates/IllustratedMessageTemplate.lit.js";
 import IllustrationMessageSize from "./types/IllustrationMessageSize.js";
 import IllustrationMessageType from "./types/IllustrationMessageType.js";
-import "./illustrations/BeforeSearch.js";
```

Comment:
This is the default illustration and we must import it, we promise this in the IllustrationMessage "name" API.

It works now this change only because "import "./dist/illustrations/AllIllustrations.js"; is imported. 
And this is a new module, and removing `import "./illustrations/BeforeSearch.js";` is a breaking change as the apps did not need to import anything to display BeforeSearch.